build-parent/pom.xml

```diff
@@ -155,6 +155,8 @@
         <revapi.skip>true</revapi.skip>
         <!-- By default don't check dependencies -->
         <revapi.checkdeps>false</revapi.checkdeps>
+        <!-- The message that is appended to the end of the plugin error output when the API problems fail the build -->
+        <revapi.buildFailureMessage>Run "jbang revapi-update" to update api-changes.xml file</revapi.buildFailureMessage>
```

Comment:
I would also suggest to the user that they need to add the justification for those changes in that file.